examples/qml_minimal/rust/src/cxxqt_object.rs

```diff
@@ -33,10 +33,10 @@ pub mod qobject {
     // ANCHOR: book_rustobj_invokable_signature
     unsafe extern "RustQt" {
```

Comment:
Tangential to this PR, but should this be combined with the above `extern "RustQt"` block?

Reply:
Yup it can, they are just separate so there are separate blocks for the book to anchors to. But yup can be combined or separate it doesn't matter.